src/datasets/arrow_dataset.py

```diff
@@ -228,114 +238,107 @@ def _get_output_signature(dataset: "Dataset", collate_fn: Callable, collate_fn_a
             batch_size (:obj:`int`): The size of batches loaded from the dataset. Used for shape inference.
```

Comment:
docstrings need to be updated here as well

Reply:
Done!